src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
```

Comment:
don't hardcode the width, what if you want only 4 or 6 stars? this will break the gap between them 🥲

Reply:
But I don't have external container, and they will take all free space(screen width) if no width added.